src/pages/GetInvolved/GetInvolved.scss

```diff
@@ -0,0 +1,57 @@
+@import "../../styles//utilities/colors.scss";
+@import "../../styles/fonts.scss";
+@import "../../styles/utilities/breakpoints.scss";
+
+.getinvolved {
+  &__title {
+    top: 216px;
+    left: 426px;
+    width: 891px;
+    height: 69px;
+    border-radius: nullpx;
+
+    font-weight: 400;
+    font-style: normal;
+    font-size: 42px;
+    font-family: Lora;
+    line-height: 48px;
+    letter-spacing: 0em;
+    text-align: left;
```

Comment:
this seems like a some overkill - lets try to standardize this a bit and reuse some of these that are getting used below.. The preview build is broken so we need that fixed so we can make adjustments to style. Design will also need to review. Could you merge main into your branch?